ps_wirepayment.php

```diff
@@ -122,13 +122,21 @@ public function uninstall()
     protected function _postValidation()
     {
         if (Tools::isSubmit('btnSubmit')) {
-            Configuration::updateValue(self::FLAG_DISPLAY_PAYMENT_INVITE,
-                Tools::getValue(self::FLAG_DISPLAY_PAYMENT_INVITE));
+            Configuration::updateValue(
+                self::FLAG_DISPLAY_PAYMENT_INVITE,
+                Tools::getValue(self::FLAG_DISPLAY_PAYMENT_INVITE)
+            );
 
             if (!Tools::getValue('BANK_WIRE_DETAILS')) {
                 $this->_postErrors[] = $this->trans('Account details are required.', [], 'Modules.Wirepayment.Admin');
             } elseif (!Tools::getValue('BANK_WIRE_OWNER')) {
                 $this->_postErrors[] = $this->trans('Account owner is required.', [], 'Modules.Wirepayment.Admin');
+            } elseif (!Validate::isUnsignedInt(Tools::getValue('BANK_WIRE_RESERVATION_DAYS'))) {
+                $this->_postErrors[] = $this->trans(
+                    'There is an invalid number of days the items remain reserved.',
```

Comment:
Let's use this existing string:
'The field is invalid. Please enter a positive integer.'
in Admin.Notifications.Error

Reply:
:warning: The message is not displayed on the bottom of the field, but on the top of the page. We should indicate the correct field.

Reply:
Ping @Julievrz

Reply:
Sorry, I missed your message. 

It would be better to have the message under the field but if that's not possible then we could have "The %field% is invalid. Please enter a positive integer."

Reply:
@Julievrz It's fixed.